addons/l10n_es_edi_tbai/models/account_move.py

```diff
@@ -188,6 +188,15 @@ def _l10n_es_tbai_lock_move(self):
     # WEB SERVICE CALLS
     # -------------------------------------------------------------------------
 
+    def l10n_es_tbai_resend_bill(self):
+        self.ensure_one()
+        self.l10n_es_tbai_post_document_id = False
+        error = self.with_context(batuz_correction=True)._l10n_es_tbai_post()
+        if error:
+            error = error + "\n\n" + _("Be careful here if you modified this vendor bill, "
+                                       "because the official version is still the old one sent. ")
```

Comment:
```suggestion
            error = error + "\n\n" + _("Be careful if you modify this vendor bill, "
                                       "because the official version is still the previous one sent.")
```
What do you think of this instead ?

Reply:
Ok, but we should keep modified as you modified it and then you sent it again and then it gave an error.